src/pages/zoeken/[slug]/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { ContentPageRenderer } from '@meemoo/admin-core-ui';
+import { HTTPError } from 'ky';
+import { GetServerSidePropsResult, NextPage } from 'next';
+import getConfig from 'next/config';
+import { useRouter } from 'next/router';
+import { GetServerSidePropsContext } from 'next/types';
+import { ComponentType, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { BooleanParam, StringParam, useQueryParams, withDefault } from 'use-query-params';
+
+import { withAdminCoreConfig } from '@admin/wrappers/with-admin-core-config';
+import { AuthModal } from '@auth/components';
+import { selectUser } from '@auth/store/user';
+import { SHOW_AUTH_QUERY_KEY, VISITOR_SPACE_SLUG_QUERY_KEY } from '@home/const';
+import { Loading } from '@shared/components';
+import { getDefaultServerSideProps } from '@shared/helpers/get-default-server-side-props';
+import { renderOgTags } from '@shared/helpers/render-og-tags';
+import { useNavigationBorder } from '@shared/hooks/use-navigation-border';
+import { selectShowAuthModal, setShowAuthModal, setShowZendesk } from '@shared/store/ui';
+import { DefaultSeoInfo } from '@shared/types/seo';
+import VisitorSpaceSearchPage from '@visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage';
+import { useGetVisitorSpace } from '@visitor-space/hooks/get-visitor-space';
+import { VisitorSpaceService } from '@visitor-space/services';
+
+import { useGetContentPage } from '../../../modules/content-page/hooks/get-content-page';
+import { ContentPageService } from '../../../modules/content-page/services/content-page.service';
+
+import { VisitorLayout } from 'modules/visitors';
+
+const { publicRuntimeConfig } = getConfig();
+
+type DynamicRouteResolverProps = {
+	title: string | null;
+} & DefaultSeoInfo;
+
+const DynamicRouteResolver: NextPage<DynamicRouteResolverProps> = ({ title, url }) => {
```

Comment:
Add a TODO to simplify this component to VisitorSpaceSearchPage after we've merged this with the changes of silke
On this route we know slug will be a visitor space, so we don't need to make a call to useGetContentPage